flink-table/flink-sql-gateway/src/main/java/org/apache/flink/table/gateway/service/materializedtable/MaterializedTableManager.java

```diff
@@ -697,8 +752,8 @@ private void validatePartitionSpec(
         if (!nonStringPartitionKeys.isEmpty()) {
             throw new ValidationException(
                     String.format(
-                            "Currently, manually refreshing materialized table only supports specifying char and string type"
-                                    + " partition keys. All specific partition keys with unsupported types are:\n\n%s",
+                            "Currently, manually refreshing materialized table only supports referring to char, varchar and string type"
```

Comment:
Could you help remove this 'manually' word? It could also be a periodic refresh here.